internal/git/test-fixtures/Makefile

```diff
@@ -1,18 +1,21 @@
 
 .PHONY: all
-all: repos/remote-repo repos/tagged-repo repos/commit-in-repo repos/tag-range-repo
+all: repos/remote-repo repos/tagged-repo repos/commit-in-repo repos/tag-range-repo repos/annotated-tagged-repo
 
-repos/remote-repo:
+repos/remote-repo: create-remote-repo.sh
```

Comment:
Makefile question: What's the point of depending on the file `create-remote-repo.sh` if nothing is going to create it?

Reply:
in case create-remote-repo.sh content changes, then the target will be run again (it's based on a mod timestamp comparison)

Reply:
oh fun! I won't do this though, seems to cause issues in CI where a cache is restored (thus the timestamps could be off)